tests/pytype_test.py

```diff
@@ -187,31 +202,28 @@ def pytype_test(args):
     running_tests = collections.deque()
     max_code, runs, errors = 0, 0, 0
     files = pytype_run + pytd_run
-    total_files = len(files)
+    total_tests = len(files)
+    # Files in stdlib/2and3 get tested twice
+    total_tests += len([f for f in pytype_run if 'stdlib/2and3' in f])
```

Comment:
You can do this in a slightly more concise way as sum(1 for f in ...).

Reply:
done